Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This Dockerfile builds a mainnet version for API3 DAO dashboard
+
+FROM node:lts-alpine as builder
+RUN apk add --no-cache git
+WORKDIR /usr/src/app
+ADD . .
+COPY src/contract-deployments/mainnet-dao.json ./src/contract-deployments/localhost-dao.example.json
+RUN yarn 
+RUN yarn tsc && yarn build
```

Comment:
You don't need `yarn tsc` as that only runs TS checks

Reply:
+